lib/components/viewers/nearby/stop.tsx

```diff
@@ -132,61 +78,21 @@ const Stop = ({
     </StyledAlert>
   )
 
-  const onMouseEnter = useCallback(() => {
-    setHoveredStop(stopData.gtfsId)
-  }, [setHoveredStop, stopData.gtfsId])
-
-  const onMouseLeave = useCallback(() => {
-    setHoveredStop(undefined)
-  }, [setHoveredStop])
-
   if (nearbyViewConfig?.hideEmptyStops && patternRows.length === 0) return <></>
 
   return (
-    <Card onMouseEnter={onMouseEnter} onMouseLeave={onMouseLeave}>
-      <CardHeader>
-        <CardTitle>
-          <IconWithText
-            icon={
-              <>
-                {transitOperators
-                  ?.filter((to) => Array.from(agencies).includes(to.agencyId))
-                  // Second pass to remove duplicates based on name
-                  .filter(
-                    (to, index, arr) =>
-                      index === arr.findIndex((t) => t?.name === to?.name)
-                  )
-                  .map((to) => (
-                    <Operator key={to.agencyId} operator={to} />
-                  ))}
-              </>
-            }
-          >
-            {stopData.name}
-          </IconWithText>
-        </CardTitle>
-      </CardHeader>
-      <CardBody>
-        <div>
-          <FormattedMessage
-            id="components.StopViewer.displayStopId"
-            values={{
-              stopId: stopData.code || stopData.gtfsId,
-              strong: Strong
-            }}
-          />
-          <button
-            className="link-button pull-right"
-            onClick={() => setViewedStop({ stopId: stopData.gtfsId }, 'stop')}
-            style={{ fontSize: 'small' }}
-          >
-            <IconWithText Icon={Calendar}>
-              <FormattedMessage id="components.StopViewer.viewSchedule" />
-            </IconWithText>
-          </button>
-        </div>
-        {fromToSlot}
-      </CardBody>
+    <Card>
+      <StopCardHeader
```

Comment:
Thoughts on adding a `titleAs="h4"` here?

Reply:
I think each card in the nearby view is part of a list already, so there is no need for headers there.